src/fragments/lib/auth/js/enable-signin-signup-signout.mdx

```diff
@@ -86,9 +152,35 @@ async function confirmSignUp() {
   }
 }
 ```
+</Block>
+</BlockSwitcher>
 
 As part of the options parameter, you can specify the `forceAliasCreation` boolean. This is set to `true` by default. If set to `false` and the email or phone number used for sign-up confirmation already exists as an alias for a different user, the API will throw an `AliasExistsException` error. If set to `true`, the API call will migrate the alias from the previous user to a newly created user.
```

Comment:
can we please remove this whole forceAliasCreation section and the code snippets related to it? It is too confusing for customers. I have checked with the team about this.

Reply:
Done